src/teleop/gui/src/components/GimbalControls.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <div class="wrap">
+        <div v-show='false' id="key">
+          <input v-on:keydown="keymonitordown">
+          <input v-on:keyup="keymonitorup">
+        </div>
+        <p>Gimbal Controls</p>
+    </div>
+</template>
+
+<script>
+
+import ROSLIB from "roslib"
+
+export default {
+
+    data() {
+    return {
+
+        keyboard_pub: null,
+
+        inputData : {
+            w_key: 0,
+            a_key: 0,
+            s_key: 0,
+            d_key: 0
+        } 
+    }
+    },
+
+    created: function () {
+        document.addEventListener('keydown', this.keymonitordown);
```

Comment:
You probably don't need these event listener declarations here since you already use v-on statements in the template section

Reply:
Having issues with v-on event listeners, keeping both for now